### DISCUSSION TITLE
cerebral degeneration

### DISCUSSION THREAD

**Discussion by ValWood** [monarch-initiative/mondo#3873] [Bigger project]
**cerebral degeneration | MONDO:0024238**

During migration from DO-MONDO We lost the following mappings to cerebral degeneration:

+ | CADASIL 2
-- | --
  | htr12
+ | hydrocephalus
  | sec71
+ | hypomyelinating leukodystrophy
  | prs1
+ | hypomyelinating leukodystrophy 10
  | SPAPYUG7.05
+ | hypomyelinating leukodystrophy 11
  | rpc40
+ | hypomyelinating leukodystrophy 12
  | SPAC16A10.03c
+ | hypomyelinating leukodystrophy 13
  | opi10
+ | hypomyelinating leukodystrophy 3
  | asc1
+ | hypomyelinating leukodystrophy 4
  | mcp60
+ | hypomyelinating leukodystrophy 6
  | nda3
+ | hypomyelinating leukodystrophy 7 with or without oligodontia and-or hypogonadotropic hypogonadism
  | rpc1
+ | hypomyelinating leukodystrophy 8 with or without oligodontia and-or hypogonadotropic hypogonadism
  | rpc2
+ | leukodystrophy
  | rnh201, SPBPB10D8.02c
+ | leukoencephalopathy with vanishing white matter
  | ala1, tif221, tif222, tif223, tif224, tif225
+ | normal pressure hydrocephalus
  | mas2

Should any be instantiated?

**Comment by nicolevasilevsky** [comment]
Hi @ValWood The formatting in Markdown is a throwing me off. Is everything in this list a disease?  Is this a mix of disease names and genes?

Can you clarify what you mean by mappings to cerebral degeneration? In DO are these children of cerebral degeneration, or is cerebral degeneration a phenotypic feature?

Thanks!

**Comment by ValWood** [comment]
You can ignore the gene names. This is just a screen grab from our DO mapping to  "cerebral degeneration"

I left then in because if there was any query about a specific disease I would be able to locate it easily at my end to check anything.

In DO all of these genes mapped to "cerebral degeneration" so  these terms must  be children of cerebral degeneration in DO.

Some of my suggestions might be  a bit off-target as I get used to how you call a disease characteristic vs a  phenotype. but the 

 Leukodystrophies are defined:

Leukodystrophies are a group of rare, progressive, metabolic, genetic diseases that affect the brain, spinal cord and often the peripheral nerves. Each type of leukodystrophy is caused by a specific gene abnormality that leads to abnormal development or destruction of the white matter (myelin sheath) of the brain.

so cerebral degeneration seems  correct  here.

**Comment by nicolevasilevsky** [comment]
Hi @ValWood 
I am still unclear.

Are you asking for Leukodystrophy to be a child of cerebral degeneration?

If yes, could you please provide evidence (like a reference to an article in PubMed)?

Thanks!

**Comment by ValWood** [comment]
I'm not familiar with any of this, only reporting lost parentage that seemed OK.

If I Google leukodystrophy

Leukodystrophies are a group of rare, progressive, metabolic, genetic diseases that affect the brain, spinal cord and often the peripheral nerves. Each type of leukodystrophy is caused by a specific gene abnormality that leads to abnormal development or destruction of the white matter (myelin sheath) of the brain.

So this seems like all leukodystrophies should be cerebral degeneration.

 leukoencephalopathy with vanishing white matter
especially sounds as though it should be, it's in the name.

However, 'cerebral degeneration' does not exist in MONDO and and 'cerebellar degeneration' is more specific. It seems that the most appropriate parent   would be 'brain disease'. Only *some* of the leukodystrophies currently have a brain disease parent. I didn't check which of the ones above, but only 23/38 of my annotations map to "brain disease". So probably this ancestor should be added to all?

Does that make sense?

**Comment by nicolevasilevsky** [comment]
Hi @ValWood I think 'brain disease' is too specific as a parent, as leukodystrophies affect the brain, spinal cord and often the peripheral nerves. It is currently a child of 'nervous system disorder'.

Is it okay to leave this as it is?

**Comment by nicolevasilevsky** [comment]
Or should we try to get other experts to weigh in? (I'm certainly not an expert in this area.) :)

**Comment by ValWood** [comment]
I'm not sure either but it feels as though it should be especially
 for leukoencephalopathy with vanishing white matter.

And considering this qualifier:
Each type of leukodystrophy is caused by a specific gene abnormality that leads to abnormal development or destruction of the white matter (myelin sheath) of the brain.

It seems that the brain disease is a major necessary component here.

For instance Parkinson's is classed as a brain disorder, but it also has peripheral nerve symptoms.

@pnrobinson 
@cmungall

**Comment by pnrobinson** [comment]
cerebral degeneration seems much more like an HPO term. It is a feature of various diseases such as Alzheimer (https://pubmed.ncbi.nlm.nih.gov/828039/). I think that if we want to have a general "grouping" term for multiple such diseases, then "dementia" would be much better. So I would vote for restructuring this domain, we should try to plan a 1-3 hour annotatathon!

**Comment by nicolevasilevsky** [comment]
I added this to our 'workshop' project board. Hopefully we'll have a workshop sometime in the new year where we can work on bigger tickets like this.
